doc/mcuboot/readme-ncs.rst

```diff
@@ -3,14 +3,26 @@
 Using MCUboot in nRF Connect SDK
 ################################
 
-See :doc:`readme-zephyr` for general information on how to integrate MCUboot with Zephyr.
+MCUboot can be used in the nRF Connect SDK in one of the following roles:
 
-The nRF Connect SDK provides additional functionality that is available when MCUboot is included.
-This functionality is implemented in the files in the :file:`modules/mcuboot` subfolder in the `sdk-nrf`_ repository.
+* As an `Immutable first-stage bootloader <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_bootloader.html#first-stage-immutable-bootloader>`_ that can perform device firmware updates to the application.
+* As an `Upgradable second-stage bootloader <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_bootloader.html#second-stage-upgradable-bootloader>`_ that can perform device firmware updates to both itself and the application.
```

Comment:
hardcoding `latest` links isn't a good idea when not looking at `latest` rendered docs. But I guess there's no other real option unless we want to introduce circular dependencies (BAD) as with nrfxlib/nrf.

Reply:
implemented links.txt and shortcuts.txt